site/content/how-to/maintenance/upgrade-apps-without-downtime.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Upgrade Applications without Downtime"
+description: "Learn how to use NGINX Gateway Fabric to upgrade applications without downtime."
+weight: 100
+toc: true
+docs: "DOCS-000"
+---
+
+{{<custom-styles>}}
+
+## Overview
```

Comment:
I think we can delete this overview since it just repeats the description, and both displayed side-by-side seems redundant.

<img width="674" alt="image" src="https://github.com/nginxinc/nginx-gateway-fabric/assets/33876974/09770379-9976-4439-b586-ddccfa150386">

Reply:
I think line #13 can be removed and the Overview and NGF functionality sections can be merged into a single Overview section